### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ermias5**

### CODE REVIEW

**Comment by ermias5** [comment]
latest

File: tailwind.config.ts
Comment (Melka1) [inline tailwind.config.ts:1]: there is a lot to unpack here,
-the name of a color can not be "bg-gray", when coloring a background it becomes,  bg-bg-gray, which is non-sense,
-consistency is key use all colors  in hex,
-name of font family can not be "nav-bar-text-21", change it,
-arrage the font sizes in ascending order and i dont get the naming process, why "mid", "lgi", "81xl", remove the inherit, us can use font-[inherit], avoid redundancy
-what the f*** is "8xs" for border radius
-the break points are redundant too, you already have "sm" for 640px, "lg" for 1024, "xl" for 1280, if you wanted smaller you could have used "xs", and i don't get the need for adding breakpoint higher than 1536px

File: components/services/services.tsx
Comment (Melka1) [inline components/services/services.tsx:1]: i dont get why you converted the p tag to flex if you wanted to center it why not use text-align:center

File: components/services/service-card-list.tsx
Comment (Melka1) [inline components/services/service-card-list.tsx:1]: remove empty className, why put the p in a div, if it does not have a style

File: app/globals.css
Comment (Melka1) [inline app/globals.css:1]: remove comments before merging, that are not useful

File: components/about-me/about.tsx
Comment (Melka1) [inline components/about-me/about.tsx:1]: -empty classname
-line 26 {"About me"}, why?
-use the tags correctly line-29 text in div

File: components/contact-section/contact.tsx
Comment (Melka1) [inline components/contact-section/contact.tsx:1]: use div if it has more than one child or for styling puposes, line 37, h1 in div, why?

**Review by Melka1 — CHANGES_REQUESTED** [review]

File: components/navigation/navbar.tsx
Comment (Melka1) [inline components/navigation/navbar.tsx:1]: line-34 don't repeat things use the conditional correctly, the default flex-direction is row no need  to specify it

**Review by Melka1 — CHANGES_REQUESTED** [review]
reply fast

File: components/navigation/navbar.tsx
Comment (Melka1) [reply components/navigation/navbar.tsx:1]: what did you resolve? flex items-center justify-center desktop:gap-0?

File: components/footer-section/footer.tsx
Comment (Melka1) [inline components/footer-section/footer.tsx:1]: remove defau;t values from this file too

**Review by Melka1 — CHANGES_REQUESTED** [review]

**Review by Melka1 — APPROVED** [review]

File: components/navigation/navbar.tsx
Comment (Melka1) [reply components/navigation/navbar.tsx:1]: line 55 too, check every thing

**Review by Melka1 — CHANGES_REQUESTED** [review]

File: components/navigation/navbar.tsx
Comment (ermias5) [reply components/navigation/navbar.tsx:1]: done!

**Review by Melka1 — APPROVED** [review]

**Pull request merged**